addons/account/views/account_menuitem.xml

```diff
@@ -38,34 +39,28 @@
-                <menuitem id="action_account_reconcile_model_menu" action="account.action_account_reconcile_model" sequence="3" groups="account.group_account_user"/>
-            </menuitem>
-            <menuitem id="account_account_menu" name="Accounting" groups="account.group_account_manager" sequence="3">
+            <menuitem id="account_account_menu" name="Accounting" groups="account.group_account_manager" sequence="1">
                 <menuitem id="menu_action_account_form" action="action_account_form" groups="account.group_account_readonly" sequence="1"/>
                 <menuitem id="menu_action_tax_form" action="action_tax_form" sequence="2"/>
                 <menuitem id="menu_action_account_journal_form" action="action_account_journal_form" groups="account.group_account_manager" sequence="3"/>
                 <menuitem id="account_report_folder" name="Reporting" groups="account.group_account_readonly" sequence="4"/>
-                <menuitem id="menu_action_currency_form" action="base.action_currency_form" name="Currencies" sequence="4"/>
-                <menuitem id="menu_action_account_fiscal_position_form" action="action_account_fiscal_position_form" sequence="5"/>
+                <menuitem id="menu_action_currency_form" action="base.action_currency_form" name="Currencies" sequence="5"/>
+                <menuitem id="menu_action_account_fiscal_position_form" action="action_account_fiscal_position_form" sequence="6"/>
                 <menuitem id="menu_action_account_journal_group_list" name="Multi-Ledger" action="action_account_journal_group_list" groups="account.group_account_readonly" sequence="7"/>
-                <menuitem id="menu_action_tax_group" action="action_tax_group" sequence="8" groups="base.group_no_one"/>
+                <menuitem id="menu_action_tax_group" action="action_tax_group" groups="base.group_no_one" sequence="8"/>
+                <menuitem id="menu_action_rounding_form_view" action="rounding_list_action" groups="account.group_cash_rounding" sequence="9"/>
             </menuitem>
-            <menuitem id="root_payment_menu" name="Online Payments" groups="account.group_account_manager" sequence="4"/>
-            <menuitem id="account_management_menu" name="Management" groups="account.group_account_manager" sequence="5">
-                <menuitem id="menu_product_product_categories" action="product.product_category_action_form" sequence="3"/>
-                <menuitem id="menu_action_rounding_form_view" action="rounding_list_action" groups="account.group_cash_rounding" sequence="4"/>
+            <menuitem id="account_invoicing_menu" name="Invoicing" groups="account.group_account_invoice,account.group_account_readonly" sequence="2">
+                <menuitem id="menu_action_payment_term_form" action="action_payment_term_form" sequence="1"/>
```

Comment:
it's usually useful to leave a gap between the sequences (in case someone / some module wants to slip something in)
maybe use 5, 10, 15... if it's not going to lead to too much of a diff in many different modules
same applies for all menuitems